packages/section-query-generator/src/index.js

```diff
@@ -69,6 +69,7 @@ module.exports = async ({
             else: '$expires',
           },
         },
+        scheduled: '$startDate',
```

Comment:
What's this intended to be used for?

Reply:
Currently there isn't a field on sectionQuery if the schedule has a startDate set prior to the content published date (why this is a "thing" beats me). This adds one.

<img width="1464" alt="Screen Shot 2022-02-15 at 2 40 29 PM" src="https://user-images.githubusercontent.com/46794001/154155796-fe7401b3-4695-4d8f-8693-61bd9ca4daac.png">
<img width="1466" alt="Screen Shot 2022-02-15 at 2 40 24 PM" src="https://user-images.githubusercontent.com/46794001/154155788-f34247c5-78de-4ef6-b407-4ea21b211157.png">

Reply:
For what purpose? Section query intentionally uses published date to correct certain schedule start dates... in that example, that start date is actually wrong... you shouldn't be able to start content before it's actually published. Nothing in the interface prevents this, per se, hence why section query enforces this

Reply:
https://trello.com/c/XpVjx5vG/277-incorrect-date-being-displayed-on-content-list-items-for-non-primary-sites